sqlmesh/core/engine_adapter/athena.py

```diff
@@ -296,14 +307,28 @@ def _build_table_properties_exp(
         return None
 
     def _truncate_table(self, table_name: TableName) -> None:
-        table = exp.to_table(table_name)
-        # Athena doesnt support TRUNCATE TABLE. The closest thing is DELETE FROM <table> but it only works on Iceberg
-        self.execute(f"DELETE FROM {table.sql(dialect=self.dialect, identify=True)}")
+        if isinstance(table_name, str):
+            table_name = exp.to_table(table_name)
```

Comment:
i don't think this one is necessary, because you handle it it in all the other functions right?

Reply:
`_truncate_table` needs this signature (or mypy will fail) because it's defined in the EngineAdapter superclass and im just overriding it